leftwm/doc/leftwm.1

```diff
@@ -0,0 +1,343 @@
+.hy
+.SH leftwm(1) Manual Page
+.SS NAME
+.PP
+LeftWM - A tiling window manager for Adventurers.
+.PP
+For comprehensive documentation of leftwm please see:
+https://github.com/leftwm/leftwm/wiki
+.SS SYNOPSIS
+.PP
+\f[B]leftwm\f[R] [SUBCOMMAND \&...]
+.SS Description
+.PP
+Starts the left window manager on $DISPLAY.
+This also boots \f[C]leftwm-worker\f[R] and logs for errors to the
+console.
+The list of available options for \f[C]leftwm\f[R] is listed below:
+.PP
+SUBCOMMAND can be:
+.PP
+\f[B]-h, \[en]help\f[R]: Prints help information.
+.PP
+\f[B]-v, \[en]version\f[R]: Prints version information.
+.PP
+\f[B]check\f[R]: Checks the syntax of the configuration file
+.PP
+\f[B]command\f[R]: Sends external commands to \f[C]leftwm\f[R]
+.PP
+\f[B]state\f[R]: Prints the current state of leftwm (in JSON format)
+.PP
+\f[B]theme\f[R]: Manage leftwm themes
+.SS Configuring leftwm
+.PP
+The settings file to change key bindings and the default mod key can be
+found at:
+.IP
+.nf
+\f[C]
+$HOME/.config/leftwm/config.toml
+\f[R]
+.fi
+.PP
+this file is automatically generated when leftwm or leftwm-check is run
+for the first time.
+.SS Default keybinds
+.PP
+\f[B]up, down, left or right refer to the arrow keys in your
+keyboard\f[R]
+.PP
+.TS
+tab(@);
+lw(21.1n) lw(48.9n).
+T{
+Keybinding
+T}@T{
+Description
+T}
+_
+T{
+Mod + (1-9)
+T}@T{
+Switch to a desktop/tag
+T}
+T{
+Mod + Shift + (1-9)
+T}@T{
+Move the focused window to desktop/tag
+T}
+T{
+Mod + W
+T}@T{
+Switch the desktops for each screen.
+Desktops [1][2] changes to [2][1]
+T}
+T{
+Mod + Shift + W
+T}@T{
+Move window to the other desktop
+T}
+T{
+Mod + (up or down)
+T}@T{
+Focus on the different windows in the current workspace
+T}
+T{
+Mod + Shift + (up or down)
+T}@T{
+Move the different windows in the current workspace
+T}
+T{
+Mod + Enter
+T}@T{
+Move selected window to the top of the stack in the current workspace
+T}
+T{
+Mod + Ctrl + (up or down)
+T}@T{
+Switch between different layouts
+T}
+T{
+Mod + Shift + (left or right)
+T}@T{
+Switch between different workspaces
+T}
+T{
+Mod + Shift + Enter
+T}@T{
+Open a terminal
+T}
+T{
+Mod + Ctrl + L
+T}@T{
+Lock the screen
+T}
+T{
+Mod + Shift + X
+T}@T{
+Exit LeftWM
+T}
+T{
+Mod + Shift + Q
+T}@T{
+Close the current window
+T}
+T{
+Mod + Shift + R
+T}@T{
+Reload LeftWM and its config
+T}
+T{
+Mod + p
+T}@T{
+Use dmenu to start application
+T}
+.TE
+.SS Floating Windows
+.PP
+You can optionally switch between tiling or floating mode for any
+window.
+.PP
+.TS
+tab(@);
+l l.
+T{
+Keybinding
+T}@T{
+Description
+T}
+_
+T{
+Mod + MouseDrag
+T}@T{
+Switch a tiled window to floating mode
+T}
+T{
+Mod + RightMouseDrag
+T}@T{
+Resize a window
+T}
+T{
+Drag window onto a tile
+T}@T{
+Switch a floating window to tiling mode
+T}
+T{
+Mod + Shift + (1-9)
+T}@T{
+Switch a floating window to tiling mode
+T}
+.TE
+.SS Configuration
+.PP
+All entries require a modifier, even if blank: \f[C]modifier = []\f[R]
```

Comment:
It might be misleading to have this info in this place. This should be mentioned inthronisiere keybinds section. 

Or is it the `Configuration` heading two lines above that is wrong?